python_ta/debug/accumulation_table.py

```diff
@@ -101,14 +104,15 @@ def _create_iteration_dict(self) -> dict:
         and loop variable to its respective value during each iteration
         """
         return {
-            "iteration": list(range(len(list(self.loop_variables.values())[0]))),
+            "iteration": list(range(len(list(self.loop_accumulators.values())[0]))),
             **self.loop_variables,
             **self.loop_accumulators,
         }
 
     def _tabulate_data(self) -> None:
         """Print the values of the accumulator and loop variables into a table"""
         iteration_dict = self._create_iteration_dict()
+        print("\n")
```

Comment:
Revert this change. In general please do not make changes that are not related to the task you're assigned.